src/main/kotlin/no/nav/familie/ef/sak/journalføring/JournalføringController.kt

```diff
@@ -43,6 +46,21 @@ class JournalføringController(private val journalføringService: Journalføring
         return Ressurs.success(journalføringService.hentDokument(journalpostId, dokumentInfoId))
     }
 
+    @GetMapping("/{journalpostId}/dokument-pdf/{dokumentInfoId}")
+    fun hentDokumentSomPdf(@PathVariable journalpostId: String, @PathVariable dokumentInfoId: String): ResponseEntity<ByteArray> {
+        val (_, personIdent) = finnJournalpostOgPersonIdent(journalpostId)
+        tilgangService.validerTilgangTilPersonMedBarn(personIdent)
+        val responseHeaders = HttpHeaders()
+        responseHeaders.contentType = MediaType.valueOf("application/pdf")
+        responseHeaders.add("Content-Disposition", "inline; filename=" + "dokument.pdf")
```

Comment:
Var det tenkt å returnere noe annet enn dokument.pdf her? :P 
```suggestion
        responseHeaders.contentType = MediaType.APPLICATION_PDF
        responseHeaders.add("Content-Disposition", "inline; filename=dokument.pdf")
```

Reply:
Jeg må sjekke om den der blir brukt - det er "viktig" at filename slutter på pdf for at nettleseren skal åpne i egen fane, hvis jeg skjønte SO rett. Men tror ikke nødvendigvis det "filnavnet" blir brukt til noe fornuftig.